wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/BurstTimestampTableViewCell.swift

```diff
@@ -17,139 +17,70 @@
         accentColor: UIColor
     ) {
-
         self.configuration = View.Configuration(
             date: message.serverTimestamp ?? Date(),
             includeDayOfWeek: context.isFirstMessageOfTheDay,
             showUnreadDot: context.isFirstUnreadMessage,
             accentColor: accentColor
         )
-        self.actionController = nil
-    }
-
-    init(configuration: View.Configuration) {
-        self.configuration = configuration
     }
 
 }
 
 final class BurstTimestampSenderMessageCell: UIView, ConversationMessageCell {
 
-    private let timestampView: ConversationCellBurstTimestampView
-    private var configuration: BurstTimestampSenderMessageCellConfiguration?
-    private var timer: Timer?
+    struct Configuration {
+        let date: Date
+        let includeDayOfWeek: Bool
+        let showUnreadDot: Bool
+        let accentColor: UIColor
+    }
 
     weak var delegate: ConversationMessageCellDelegate?
     weak var message: ZMConversationMessage?
     weak var actionController: ConversationMessageActionController?
 
-    override init(frame: CGRect) {
-        self.timestampView = ConversationCellBurstTimestampView()
-        super.init(frame: frame)
-        configureSubviews()
-        configureConstraints()
-    }
-
-    @available(*, unavailable)
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init?(coder aDecoder: NSCoder) is not implemented")
-    }
-
-    private func configureSubviews() {
-        addSubview(timestampView)
-    }
-
-    private func configureConstraints() {
-        timestampView.translatesAutoresizingMaskIntoConstraints = false
-
-        NSLayoutConstraint.activate([
-            timestampView.leadingAnchor.constraint(equalTo: leadingAnchor),
-            timestampView.topAnchor.constraint(equalTo: topAnchor),
-            timestampView.trailingAnchor.constraint(equalTo: trailingAnchor),
-            timestampView.bottomAnchor.constraint(equalTo: bottomAnchor)
-        ])
-    }
-
-    override func willMove(toWindow newWindow: UIWindow?) {
-        super.willMove(toWindow: newWindow)
-
-        if window == nil {
-            stopTimer()
-        }
-    }
-
-    func willDisplay() {
-        startTimer()
-    }
-
-    func didEndDisplaying() {
-        stopTimer()
-    }
-
-    private func reconfigure() {
-        guard let configuration else {
-            return
-        }
-        configure(with: configuration, animated: false)
-    }
-
-    private func startTimer() {
-        stopTimer()
-        timer = Timer.scheduledTimer(withTimeInterval: 1, repeats: true) { [weak self] _ in
-            self?.reconfigure()
-        }
-    }
-
-    private func stopTimer() {
-        timer?.invalidate()
-        timer = nil
-    }
-
-    // MARK: - Cell
-
     var isSelected: Bool = false
 
-    func configure(with object: BurstTimestampSenderMessageCellConfiguration, animated: Bool) {
-        configuration = object
+    func willDisplay() {}
+    func didEndDisplaying() {}
```

Comment:
question: do we still need these methods?

Reply:
No, thanks, removed them.